File: multipay-manual-payment-microservice-api/src/Multipay.Manual.Payment.Microservice.Api.Domain/SeedWork/Paging/Paging.cs
namespace Multipay.Manual.Payment.Microservice.Api.Domain.SeedWork.Paging;

public class Paging : IPaging
{
    public int Total { get; set; }
    public int CurrentPage { get; set; }
    public int PerPage { get; set; }
    public int Pages { get; set; }
}

File: LAB-multipay-admin-site-web/src/domain/seedWork/paging/Paging.ts
import IPaging from './IPaging'

export class Paging implements IPaging {
  currentPage: number
  pages: number
  perPage: number
  total: number

  constructor(
    currentPage: number = 1,
    pages: number = 0,
    perPage: number = 10,
    total: number = 0
  ) {
    this.currentPage = currentPage
    this.pages = pages
    this.perPage = perPage
    this.total = total
  }
}
